exporter/collstats_collector.go

```diff
@@ -77,7 +77,14 @@ func (d *collstatsCollector) collect(ch chan<- prometheus.Metric) {
 		collections = fromMapToSlice(namespaces)
 	}
 
-	for _, dbCollection := range collections {
+	onlyCollections, err := filterCollectionsWithoutViews(d.ctx, client, collections)
```

Comment:
I think in case someone passed view in the CLI input we shouldn't filter it and just show error messages like we used to. it's users fault and we shouldn't try to do extra job for him.